src/DataTable/__tests__/DataTable.test.tsx

```diff
@@ -0,0 +1,201 @@
+import React from 'react'
```

Comment:
Is there a reason you put this in a `__tests__` directory?

Reply:
@colebemis was anticipating having other tests grouped under here for the different features / sub-components

Reply:
Sounds good 👍